go.mod

```diff
@@ -64,13 +64,14 @@ require (
 	github.com/shirou/gopsutil/v3 v3.22.12
 	github.com/shopspring/decimal v1.3.1
 	github.com/smartcontractkit/caigo v0.0.0-20230530082629-53a5a4bdb25e
-	github.com/smartcontractkit/chainlink-cosmos v0.4.1-0.20230525203711-20bed74ac906
-	github.com/smartcontractkit/chainlink-relay v0.1.7-0.20230531014621-9c303da4c086
-	github.com/smartcontractkit/chainlink-solana v1.0.3-0.20230531071550-c058f7c3964f
-	github.com/smartcontractkit/chainlink-starknet/relayer v0.0.0-20230531021926-052aa18e1b0e
-	github.com/smartcontractkit/libocr v0.0.0-20230525150148-a75f6e244bb3
-	github.com/smartcontractkit/ocr2keepers v0.6.15
-	github.com/smartcontractkit/ocr2vrf v0.0.0-20230510102715-c58be582bf19
+	github.com/smartcontractkit/chainlink-cosmos v0.4.1-0.20230531184247-9617be7da020
+	github.com/smartcontractkit/chainlink-relay v0.1.7-0.20230531175356-03a8f54e3528
+	github.com/smartcontractkit/chainlink-solana v1.0.3-0.20230531175652-9edfbba8ed45
```

Comment:
It looks like this solana hash is an open draft?
https://github.com/smartcontractkit/chainlink-solana/pull/503

Reply:
Yes I will aim to get all supporting PRs merged first then this one